code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -43,23 +43,23 @@
 	var/mob/living/carbon/human/friend
 	var/tamed = 0
 
-/mob/living/simple_animal/otie/fluff
+/mob/living/simple_animal/otie/fluff //gets the pet2tame feature. starts out hostile tho so get gamblin'
```

Comment:
Oh I see what is going on here. /fluff directories should not be handled like this. Fluff denotes a custom version of something, usually for a specific player.